### DISCUSSION TITLE
Anyone has loaded Town11?

### DISCUSSION THREAD

**Discussion by anyezhou1995** [carla-simulator/carla#5243] [Q&A]
Why I tried to use Town11 I got a blank world with only sea and sky. I used the command: world = client.load_world('Town11'). Also tried the config.py file to specify the map.
Any clues on this issue?

**Comment by KavuSlayer** [comment]
You must spawn ego vehicle to make the map load.
Since it is a tiled map, the tiles is loaded with the vehicle moving through them.

**Reply by anyezhou1995** [reply]
Can you explain more detail? We need to load the map first then get a spawn point to spawn the ego vehicle, right? How can we spawn ego vehicle first then load the map?

**Reply by SMaoIM** [reply]
> Can you explain more detail? We need to load the map first then get a spawn point to spawn the ego vehicle, right? How can we spawn ego vehicle first then load the map?

Hello, have you found any solutions?

**Reply by Miknow78** [reply]
Hello, did u find a solution??

**Comment by Nitro60zeus** [comment]
> Why I tried to use Town11 I got a blank world with only sea and sky. I used the command: world = client.load_world('Town11'). Also tried the config.py file to specify the map. Any clues on this issue?

Once you open the map (with the blank world), run manual_control.py. 

Alternatively, you can open town11 in ue4editor, and play as standalone. You can then roam around as a spectator.
